gluco-check-common/strings/en/assistant-responses.yaml

```diff
@@ -48,3 +49,18 @@ assistant_responses:
   
   pump_battery: >
     The pump's battery is at {{percent}}%.
+
+  errors: 
```

Comment:
How do you feel about "I" vs "we" here? In general I would say that "we" instead of "I" would be more idiomatic in the context of "english in general," but if this is correct in the context of "how an assistant app talks" then "I" seems fine to me!

Reply:
That's a good point, had not occurred to me at all.
I did some quick research, but it looks like most Assistant apps talk in a 'first person singular':

- Todoist says: "I can create or read your tasks to you"
- Headspace says: "I didn't understand what you said" on errors
- wikiHow says: "Ask me how to do anything"

It does make sense in my (admittedly limited) knowledge of English because when talking to Google, you're addressing your personal (AI) assistant, not a team.

Do you have any examples of apps that talk in first person plural?  
Otherwise I think it's best to follow along with what the others are doing.

Reply:
Ah okay, that's my relative inexperience with assistant showing. Greatly favour doing what others are doing!